src/package_json.rs

```diff
@@ -66,6 +66,27 @@ mod tests {
         let expected = r###"{
   "name": "tester",
   "version": "1.2.3-rc.4"
+}"###
+            .to_string();
+        assert_eq!(std::fs::read_to_string(file).unwrap(), expected);
+    }
+
+    #[test]
+    fn test_retain_property_order() {
```

Comment:
I assume my prefixing of functions with `test_` is just a habit from Python and not required here. We may be able to omit that?

Reply:
Yeah, I believe it's redundant.  We already know it's a test, so the `test_` prefix isn't adding any value.

@dbanty Do you want me to remove that in this PR, or would you rather we remove all of them in a separate PR?

Reply:
I think we should address them as we encounter them, so swap it out here before we merge.